CONTRIBUTING.md

```diff
@@ -1,59 +1,46 @@
+We follow the [GitHub Flow](https://guides.github.com/introduction/flow/index.html), so all code changes happen through pull requests. Here’s how you can contribute:
 
-In short, when you submit code changes, your submissions are understood to be under the same [license](LICENSE) that covers the project. Feel free to contact the maintainers if that's a concern.
+1. Fork the repo and create your branch from `main`.
+2. If you've added code, please add tests. This is crucial for ensuring the integrity of the application.
+3. Ensure the test suite passes and your code adheres to the coding standards detailed below.
+4. Submit your pull request!
 
-## Report bugs using Github's [issues](https://github.com/yourusername/yourprojectname/issues)
+## Specific Technologies
 
-We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/SaintAngeLs/distributed_minispace/issues/new); it's that easy!
+Since our project is built with .NET and specifically utilizes Blazor, Radzen, and MudBlazor for the front-end:
 
-## Write bug reports with detail, background, and sample code
+- **Blazor**: Ensure your contributions are compatible with the latest stable version of Blazor.
+- **Radzen**: Contributions using Radzen components should follow the best practices recommended by Radzen documentation.
+- **MudBlazor**: Use MudBlazor for designing responsive elements and UI components. Adhere to MudBlazor's guidelines for consistency across the application.
 
-**Great Bug Reports** tend to have:
+## Coding Style
 
-- A quick summary and/or background
-- Steps to reproduce
-  - Be specific!
-  - Give sample code if you can.
-- What you expected would happen
-- What actually happens
-- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+- **.NET and C#**: Follow the [C# Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions).
+- **Blazor**: Code should be clean, well-commented, and organized within the appropriate structure.
+- **Radzen and MudBlazor**: Ensure components and services are properly configured and utilized according to the framework's standards.
 
-People *love* thorough bug reports.
+## Bug Reporting and Pull Requests
 
-## Use a Consistent Coding Style
+- Report bugs through [GitHub issues](https://github.com/SaintAngeLs/distributed_minispace/issues).
+- Write detailed bug reports with a clear background and steps to reproduce.
+- Pull requests should be clear and concise, mentioning the intent of changes and any related issues.
 
-* You can use `clang-format` for C and C++.
-* Make sure to follow [PEP8](https://pep8.org/) for Python.
-* For other languages, provide a linting configuration.
+## Licensing
 
-## License
+Any contributions you make will be under the same [MIT License](LICENSE) that covers the project, ensuring that MiniSpace remains open and free for the community.
```

Comment:
I'm not sure about it but I think we use Apache License and its files are named "LICENCE" and the link doesn't work.

Reply:
Now the link to licence should work properly.

Reply:
Yeap, the Apache licence is required there.